docs/api/topics/ratings.rst

```diff
@@ -21,6 +21,11 @@ _`List`
 
     :query app: the ID or slug of the app whose ratings are to be returned.
     :query user: the ID of the user or `mine` whose ratings are to be returned.
+    :query lang: a language to filter ratings by if `match_lang` is set.
+    :query match_lang: a boolean to specify to match language or not. If unset
+                       all results are returned, if 1 only results matching
+                       `lang` are returned if 0 only results not matching
```

Comment:
I think this only supports the `match_lang=1` case. I'm not sure if we need the inverse case, my original mockup of the UI had a "my language / not my language" control but we probably really want a "my language / all" control which I think it why this no longer supports `match_lang=0`.